packages/build/src/release.ts

```diff
@@ -1,93 +1,173 @@
-import os from 'os';
-import Config from './config';
+import { Octokit } from '@octokit/rest';
+import semver from 'semver';
 import compileExec from './compile-exec';
+import Config from './config';
+import uploadDownloadCenterConfig from './download-center';
 import uploadArtifactToEvergreen from './evergreen';
-import zip from './zip';
-
-// import { Octokit } from '@octokit/rest';
-// import releaseToGithub from './github';
-// import uploadArtifactToDownloads from './upload-artifact';
-// import uploadDownloadCenterConfig from './download-center';
-// import publishMacOs from './macos-sign';
-// import Platform from './platform';
-// import S3 from 'aws-sdk/clients/s3';
+import { GithubRepo } from './github-repo';
+import uploadArtifactToDownloadCenter from './upload-artifact';
+import { zip, ZipFile } from './zip';
+import Platform from './platform';
```

Comment:
leaving this file to only do: zipping of the whole artifact, and uploading it to downloads centre if necessary.